.github/workflows/docker-publish-huggingface.yml

```diff
@@ -0,0 +1,69 @@
+name: Build and push HuggingFace TGI docker image
+
+on:
+  workflow_dispatch:
+    inputs:
+      tgi-version:
+        description: 'tgi version'
+        required: true
+        default: '0.5.0'
+
+jobs:
+  create-runner:
+    runs-on: [ self-hosted, scheduler ]
+    steps:
+      - name: Create new CPU instance
+        id: create_cpu
+        run: |
+          cd /home/ubuntu/djl_benchmark_script/scripts
+          token=$( curl -X POST -H "Authorization: token ${{ secrets.ACTION_RUNNER_PERSONAL_TOKEN }}" \
+          https://api.github.com/repos/deepjavalibrary/djl-serving/actions/runners/registration-token \
+          --fail \
+          | jq '.token' | tr -d '"' )
+          ./start_instance.sh action_cpu $token djl-serving
+    outputs:
+      cpu_instance_id: ${{ steps.create_cpu.outputs.action_cpu_instance_id }}
+
+  build-and-push-image:
+    runs-on: [ self-hosted, cpu ]
+    timeout-minutes: 180
+    needs: create-runner
+    env:
+      TGI_VERSION: ${{github.event.inputs.tgi-version}}
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          repository: huggingface/text-generation-inference
+          ref: v${{ env.TGI_VERSION }}
+      - uses: actions/checkout@v3
```

Comment:
probably you need to clean up previous build containers. Check how we do in sync job

Reply:
Done